apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-steps.tsx

```diff
@@ -63,30 +65,67 @@ function getTourSteps(
 	isSiteEditor = false,
 	themeName: string | null = null
 ): WpcomStep[] {
+	const completedFlow = getQueryArg( window.location.href, 'completedFlow' );
 	const isVideoMaker = 'videomaker' === ( themeName ?? '' );
+	const isPatternAssemblerFlow = 'pattern_assembler' === completedFlow;
```

Comment:
Hmm, can we simplify this whole thing and just check whether `themeName` is `blank-canvas-3`? Because if this theme was selected, it means the user went to the PA flow. Then we don't need to set up new query arg. Does this work?

Reply:
Do you think it's safe enough? I'm also happy with that but the user will land on the site editor on the mobile for now. Do we want to show the new slide to mobile users?

Reply:
> Do we want to show the new slide to mobile users

Oh, I see... I forgot that we disable it for mobile users 🤦 ok then, I'll continue reviewing this approach...

Reply:
Just realized, wasn't the discussed solution (a few weeks ago) to create a new "intent" for pattern assembler (e.g. `pattern-assember`)? Then we can use `useSiteIntent()` in the code. Do we want to do that?

Reply:
> The Welcome Tour only shows automatically ONE TIME.  The first time the user/site enters the editor.

Referring to https://github.com/Automattic/wp-calypso/issues/70386#issuecomment-1334318356, we need a way to show the congrats slide **ONE TIME** so I don't introduce a new intent in this PR. Do you think we still need to introduce a new intent for now?

Reply:
OH I SEE! Now it all makes sense 😂 👍